uadapy/plotting/plots2D.py

```diff
@@ -37,9 +37,12 @@ def plot_samples(distributions, n_samples, seed=55, **kwargs):
 
     if isinstance(distributions, Distribution):
         distributions = [distributions]
-    for d in distributions:
+
+    colors = utils.get_colors(len(distributions))
```

Comment:
Could you please add the colorblind option analog to 1D? I currently don't see a reason not to do it.